fbc/Makefile

```diff
@@ -47,9 +46,7 @@ fbc-onboarding: fbc-onboarding-deps clean
 #
 # --- SEMVER TEMPLATE ---
 #catalog: semver
-#
-# --- COMPOSITE TEMPLATE ---
-catalog: composite
```

Comment:
With the removal of the composite as a default target for `make catalog` we should make the basic the default now.